src/ReefGuideAPI.jl

```diff
@@ -136,8 +136,16 @@ function setup_regional_data(config::Dict)
         @debug "Using previously generated regional data store."
     elseif isfile(reg_cache_fn)
         @debug "Loading regional data cache from disk"
-        @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
-    else
+        # updates to packages like DiskArrays can break deserialization
```

Comment:
Little nitpick, I prefer sentence case for comments that are not in line

```suggestion
        # Updates to packages like DiskArrays can break deserialization
```